tensorflow_addons/activations/rrelu.py

```diff
@@ -14,18 +14,22 @@
 # ==============================================================================
 
 import tensorflow as tf
-from tensorflow_addons.utils.types import Number
-from tensorflow_addons.utils import types
-from typing import Optional
+from tensorflow_addons.utils.types import TensorLike, Number
+
+try:
+    tf.no_gradient("StatefulUniform")
```

Comment:
I believe this deserves a comment as it's not obvious at all what is the purpose of this line.

Reply:
Yes, I'm curiously the op doesn't register the gradient.

Reply:
I'm not sure I understand. You mean the tests will fail if this line is not used?

Reply:
I test it with latest tf version and it seems the op has registered the gradient and I remove the ```tf.no_gradient```.